src/bindings/python/tests/test_graph/test_manager.py

```diff
@@ -8,17 +8,26 @@
 import numpy as np
 import pytest
 
-import openvino.runtime.opset8 as ov
-from openvino.runtime import Model
+import openvino.runtime.opset8 as ops
+from openvino.runtime import Core, Model
```

Comment:
```suggestion
import openvino.runtime.opset10 as ops
```
Bump the opset?

Reply:
I don't think we have opset11 yet https://github.com/openvinotoolkit/openvino/tree/master/src/core/include/openvino/opsets

Reply:
good catch!